AsyncioScopeManager based on contextvars and supporting Tornado 6 (#118)
opentracing/opentracing-python: testbed/test_subtask_span_propagation/README.md

Implementation details:
- For `threading`, `gevent` and `asyncio` the `Span` is manually passed down the call chain, being manually reactivated it in each corotuine/task.

> - For `tornado` and `contextvars`, the active `Span` is not passed nor activated down the chain due to implicitly context propagating.

Reviewer: ```suggestion
- For `tornado` and `contextvars`, the active `Span` is not passed down the chain nor activated because the context is implicitly propagated.
```
